### PR TITLE
Add 0 invoice question to FAQ

### LINKED ISSUE #587: Explain 0 Invoice
Add FAQ and possibly reference in Invoices.md why a 0 invoice goes to paid immediately and use case explanation.

### PR DESCRIPTION
**Pull request by britttttk**
Closes #587

### CODE REVIEW

File: docs/FAQ/FAQ-Stores.md [hunk 1]
```diff
@@ -24,6 +25,10 @@ To create your first store, go to > Stores from the header menu and click "creat
 
 There's no limit on a number of stores you can create in BTCPay.
 
+## why are invoices without payment showing as complete?
```
Comment (pavlenex) [inline docs/FAQ/FAQ-Stores.md:28]: Capital W in `why`

File: docs/FAQ/FAQ-Stores.md [hunk 2]
```diff
@@ -24,6 +25,10 @@ To create your first store, go to > Stores from the header menu and click "creat
...
+
+When an invoice is created to receive a payment value of 0 (zero amount due) the invoice by definition, is already paid. The invoice will appear as complete as soon as it's created.
```
Comment (pavlenex) [inline docs/FAQ/FAQ-Stores.md:30]: It would be good to give more context on why this happens (giveaway, etc)

Comment (britttttk) [reply docs/FAQ/FAQ-Stores.md:30]: True, ok let me know what you think of this..

**Comment by britttttk** [comment]
this pr doesn't show my commits I don't know why. closing it

**Pull request closed without merging**